service/discovery/azure/compute.go

```diff
@@ -329,16 +329,23 @@ func (d *azureComputeDiscovery) discoverSlots(baseSite *armappservice.Site, c ar
 			return
 		}
 		for _, slot := range page.Value {
```

Comment:
Isn't that the same armappservice.Site object as in the discoverFunctionsWebApps()? If so, that shouldn't be called twice.